api.md

```diff
@@ -431,12 +434,72 @@ PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [`chunk_relation_size`](#chunk_relation_size)
 function.
 
-**Space partitions**: In most cases, it is advised for users not to use
+**Space partitions:** In most cases, it is advised for users not to use
 space partitions. The rare cases in which space partitions may be useful
 are described in the [add dimension][] section.
 
 ---
 
+## create_distributed_hypertable() [](create_distributed_hypertable)
+
+A specialized implementation of [`create_hypertable`](#create_hypertable)
+designed for creating hypertables distributed across multinode 
+environments.  The required and optional arguments are the same as for
+`create_hypertable`, as is the returned value.  The main difference is
+that this command will default the `replication_factor` optional
+parameter to a value of 1.
+
+#### Sample Usage [](create_distributed_hypertable-examples)
+
+Create a table `conditions` spread across the entire distributed
+database.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time');
+```
+
+Create a table `conditions` which will be partitioned across data
+nodes by the 'location' column.  Note that the number of space
+partitions is automatically equal to the number of data nodes in this
+case.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time', 'location');
+```
+
+Create a table `conditions` using a specific set of data nodes.  Each
+chunk will be written to two data nodes.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time', 'location', replication_factor => 2,
+    data_nodes => '{ "data_node_1", "data_node_2", "data_node_4", "data_node_7" }');
+```
+
+#### Best Practices [](create_distributed_hypertable-best-practices)
+
+**Space partitions:** As opposed to the normal [`create_hypertable` best practices]
+(#create_hypertable-best-practices), space partitions are highly
+recommended for distributed hypertables.  Incoming data will be
+divided among data nodes based upon the space partition (the first one
+if multiple space partitions have been defined).  However, if there is
+no space partition, all the data for a time slice will be written to a
+single data node, negating much of the benefit of distributing the
+hypertable.
+
+**Time intervals:** Follow the same guideline in setting the `chunk_time_interval`
+as with [`create_hypertable`](#create_hypertable-best-practices),
+bearing in mind that the calculation needs to be based on the memory
+capacity of the data nodes.  However, one additional thing to
+consider, assuming space paritioning is being used, is that the
+hypertable will be evenly distributed across the data nodes, allowing
+a larger time interval.  In this case you can consider the total
+memory across all the data nodes, though the incoming rate should also
+be multiplied by the `replication_factor` if greater than 1.
+
+If space partioning is not being used, the `chunk_time_interval`
```

Comment:
Haha, space partitioning is a struggle! Spelling nit.

Reply:
For some reason my fingers always trip up on this word, fixed.